src/igvfd/schemas/treatment.json

```diff
@@ -0,0 +1,206 @@
+{
+    "title": "Treatment",
+    "id": "/profiles/treatment.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "amount",
+        "amount_units",
+        "treatment_term_id",
+        "treatment_term_name",
+        "treatment_type"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/notes"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        }
+    ],
+    "dependentSchemas": {
+        "amount": {
+            "required": [
+                "amount_units"
+            ],
+            "comment": "amount_units must be specified if amount is specified"
+        },
```

Comment:
This dependency can be dropped as it is redundant to the required field above.  It is like saying I require childcare every day of the week AND if it is Thursday, I also require childcare.  If you drop lines 45-50 then you will also be able to drop the test below.

Reply:
I love the analogy. Made me smile 😃